packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -111,6 +112,11 @@ function toTree(vnode) {
     }
     case HostText: // 6
       return node.memoizedProps;
+    case Fragment: // 10
+    case Mode: // 11
+    case ContextProvider: // 13
```

Comment:
O.o ContextProvider and ContextConsumer are new node types, they're not just components?